app/components/ui/Cards/OppEventCard.module.scss

```diff
@@ -0,0 +1,93 @@
+@import "../../../styles/utils/_helpers.scss";
+
+.card {
+  display: flex;
+  background: #ffffff;
+  border: 2px solid var(--tailwind-gray-100, #f3f4f6);
+  border-radius: var(--rounded-md, 6px);
+  width: 100%;
+  overflow: hidden;
+  position: relative;
+  cursor: pointer;
+
+  &.opportunity {
+    flex-direction: column;
+  }
+
+  @media screen and (max-width: $break-tablet-l) {
+    flex-direction: column;
+  }
+
+  .cardImage {
+    height: auto;
+    width: 100%;
+    object-fit: cover;
+
+    &.event {
+      max-width: 40%;
+
+      @media screen and (max-width: $break-tablet-l) {
+        max-width: none;
+      }
+    }
+  }
+
+  .content {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    gap: 16px;
+    padding: 24px;
+    height: 100%;
+    width: 100%;
+
+    @media screen and (max-width: $break-tablet-p) {
+      padding: 16px;
+    }
+
+    .contentTitle {
+      font-size: 20px;
```

Comment:
A lot of the font styling gets set in _helpers.scss. Mind removing the styling you added that gets set there? Just want to try to keep the 'theme' functionality!